assertsprocessor/service_queue.go

```diff
@@ -0,0 +1,63 @@
+package assertsprocessor
+
+import (
+	"math"
+	"sync"
+)
+
+type serviceQueues struct {
+	config        *Config
+	requestStates *sync.Map
+	requestCount  int
```

Comment:
`requestStates` is a `map`. Why not use map's `len` function instead of having a separate variable to track the map size?

Reply:
See https://github.com/golang/go/issues/20680. 

Guess a len function on a current map is not preferred by the library developer. Java ConcurrentHashMap has the same concern about performance.